src/app.py

```
@@ -0,0 +1,435 @@
+            return
+        
+        if not methods:
+            messagebox.showwarning("Предупреждение", "Пожалуйста, выберите хотя бы один метод!")
+            return
+        
+        file_to_compare = self.file2_path
+        temp_file_path = None
+        
+        if self.add_noise_var.get() and self.available_noise_types:
+            try:
+                # Файлы с шумом будут сохраняться в другой файл
```

Ruff (0.14.8)

Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)